### ISSUE TITLE
Got 80.5 mAP on Pascal VOC 2007

### ISSUE THREAD

**Issue by ruoqianguo** [jwyang/fpn.pytorch#20]
hi guys, i changed the stride  in  [resnet.layer4](https://github.com/jwyang/fpn.pytorch/blob/23bd1d2fa09fbb9453f11625d758a61b9d600942/lib/model/fpn/resnet.py#L117) from 1 to 2, then i trained the model on
the union set of VOC 2007 trainval and VOC 2012 trainval (“07+12”) and evaluate on
VOC 2007 test set, i got 80.5 mAP. Besides,  i trained the model on VOC 2007 trainval and evaluate on
VOC 2007 test set, i got 75.7 mAP. You can see the details in the [repository](https://github.com/guoruoqian/FPN_Pytorch)
